src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -91,6 +92,11 @@ public class RazorViewEngineOptions
         public IList<string> AreaViewLocationFormats { get; } = new List<string>();
 
         /// <summary>
+        /// Gets the sequence of <see cref="MetadataReference" /> instances that should also be included in Razor compilation.
```

Comment:
"also" says very little. Suggest e.g. `The <see cref="MetadataReference" /> instances that should be included in Razor compilation, along with those found using <see cref="AssemblyPart" />s.`